### PR TITLE
Solution attempt

### PR DESCRIPTION
**Pull request by Likh-Alex**

### CODE REVIEW

File: src/main/java/core/basesyntax/BuilderTestApp.java [hunk 1]
```diff
@@ -1,7 +1,15 @@
...
+    public static void main(String[] args) {
+        Plane firstPlane = new Plane.PlaneBuilder()
+                .setModel("Yak 40")
+                .setCapacity(4)
+                .setType("Passenger")
+                .setYear(2010)
+                .setRange(200)
+                .build();
+
+        System.out.println(firstPlane.toString());
```
Comment (KseniiaMakarova) [inline src/main/java/core/basesyntax/BuilderTestApp.java]:
```suggestion
        System.out.println(firstPlane);
```
you can write like this as well

File: src/main/java/core/basesyntax/Plane.java [hunk 2]
```diff
@@ -1,5 +1,70 @@
...
+    }
+
+    @Override
+    public String toString() {
+        return "Plane{"
+                + "model='" + model + '\''
+                + ", capacity='" + capacity
+                + '\''
+                + ", type='"
+                + type + '\''
```
Comment (KseniiaMakarova) [inline src/main/java/core/basesyntax/Plane.java]:
```suggestion
                + ", capacity='" + capacity + '\''
                + ", type='" + type + '\''
```
same for other fields

**Review by KseniiaMakarova — CHANGES_REQUESTED** [review]

**Review by dmytro-hryhoruk — APPROVED** [review]
good job

File: src/main/java/core/basesyntax/BuilderTestApp.java (hunk 1 above)
Comment (Likh-Alex) [reply src/main/java/core/basesyntax/BuilderTestApp.java]: Got it, done

File: src/main/java/core/basesyntax/Plane.java (hunk 2 above)
Comment (Likh-Alex) [reply src/main/java/core/basesyntax/Plane.java]: fixed

**Pull request closed without merging**